docs/API.md

```diff
@@ -1214,6 +1214,21 @@ sender and can be useful in situations where it is important to know whether a
 connection is possible. The inability to decrypt an incoming transmission can
 be then reported as a different kind of problem.
 
+**IMPORTANT**: There is an unusual behavior in case when this flag is TRUE on
+the caller and FALSE on the listener. This is because the listener accepts the
```

Comment:
```suggestion
**IMPORTANT**: There is unobvious behavior in the case when this flag is TRUE on
the caller and FALSE on the listener. If peers have a mismatch in the passphrases set,
for the listener side the connection will be established and broken right after.
This is because the listener accepts the
```

Reply:
Ah, ok, I'll improve the suggestion in the Steve's comment.